MapboxNavigation/Error.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+extension NSError {
+    /**
+     Creates a custom `Error` object.
+     */
+    public static func withCustom(message: String, errorCode: SpokenVoiceError = .defaultError) -> NSError {
+        return NSError(domain: "MBErrorDomain", code: errorCode.rawValue, userInfo: [
```

Comment:
Pull this domain out into a global constant, along the lines of the map SDK’s `MGLErrorDomain`.